doc/man3/BIO_read.pod

```diff
@@ -91,7 +92,7 @@ See L<BIO_should_retry(3)> for details of how to
 determine the cause of a retry and other I/O issues.
 
 If the "gets" method is not supported by a BIO then BIO_get_line() can be used.
-In this case it is also possible to use BIO_gets()
+In this case it is also possible to make BIO_gets() usable
```

Comment:
This is still slightly confusing as at first sight it is not clear what the `In this case` refers to. When I read it first it was unclear tp me why I would use BIO_gets when I would use BIO_get_line(). Only then I realized it refers to the condition.

Reply:
Ah, so I've made it more explicit:
```
It is also possible to make BIO_gets() usable even if the "gets" method is not
supported by adding a buffering BIO L<BIO_f_buffer(3)> to the chain.

Reply:
Or would you prefer the following?
```
It is also possible to make BIO_gets() more generally usable 
by adding a buffering BIO L<BIO_f_buffer(3)> to the chain.

Reply:
It is fine as is now. Thanks.